assets/src/scss/layout/navbar/_burger-menu.scss

```diff
@@ -1,3 +1,20 @@
+@mixin slide-from-start() {
```

Comment:
Going back to the previous question, what's the motivation to add this `mixin`? Is only used here and only one time

Reply:
https://github.com/greenpeace/planet4-master-theme/pull/1673#discussion_r850512997

Reply:
TBH, if it's moved to this file it doesn't make sense to use a `mixin`

Reply:
In this case it's used more than once, just mean that it's common to put a mixin in a file if that makes it more readable.

Reply:
We can move this later once we have a second use case.